internals/webpack/client/webpack.base.babel.js

```diff
@@ -59,7 +59,7 @@ module.exports = (options) => ({
         ],
       },
       {
-        test: /\.(eot|svg|otf|ttf|woff|woff2)$/,
+        test: /\.(jpg|eot|svg|otf|ttf|woff|woff2)$/,
```

Comment:
@nastyakitsune This is the changed that required image import to start working. I guess to finalize it we need to double-check all configs (dev, prod) and synchronize the loaders.

Reply:
Ok, let's add also `jpeg|png`